jmx_prometheus_javaagent/src/main/java/io/prometheus/jmx/JavaAgent.java

```diff
@@ -39,6 +41,7 @@ public class JavaAgent {
     private static final String DEFAULT_HOST = "0.0.0.0";
 
     private static HTTPServer httpServer;
+    private static OpenTelemetryExporter openTelemetryExporter;
```

Comment:
These variables are not used, they can be removed.

Reply:
I was unsure of the GC behavior if the objects weren't referenced. I'll remove them.